client/src/relatedProducts/RelatedProductsView/RelatedProducts.jsx

```diff
@@ -42,6 +42,7 @@ export default class RelatedProducts extends React.Component {
   }
 
   getRelatedStateData() {
+    console.log('hi')
```

Comment:
Think I can axe this function altogether now with new and improved functions for getting data!

Reply:
🔥 im in agreement